packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.tsx

```diff
@@ -164,7 +164,17 @@ export class SceneTreeTableCell {
               )}
             </button>
           )}
-          <div class="content">{this.displayValue()}</div>
+          {this.value && (
```

Comment:
You should use `this.value != null` here. An empty string will evaluate to `false` otherwise.